cosmwasm/packages/wasmi-runtime/Enclave.config.xml

```diff
@@ -3,7 +3,7 @@
   <ProdID>0</ProdID>
   <ISVSVN>0</ISVSVN>
   <StackMaxSize>0x800000</StackMaxSize>
-  <HeapMaxSize>0x4000000</HeapMaxSize>
+  <HeapMaxSize>0x8000000</HeapMaxSize>
```

Comment:
Can we increase to 256 to be extra safe?

Reply:
sure. Do you want to adjust the recursion limit to 10 as well, or do you want to have a huge margin of error?

Reply:
Huge margin of error:)

Reply:
cool :)

Reply:
@Cashmaney will 256 MiB work in the CIs?